pipeline/inferrer/feature_inferrer/app/main.py

```diff
@@ -1,3 +1,4 @@
+import numpy as np
```

Comment:
nit: this is a third-party import, should be between fastapi and weco_datascience